.travis.yml

```diff
@@ -21,4 +20,4 @@ script:
     - vendor/bin/tester -p php $COVERAGE tests/
 
 after_script:
-    - if [ $TRAVIS_PHP_VERSION = '5.6' ]; then wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover coverage.xml; fi
+    - if [ $TRAVIS_PHP_VERSION = '5.6' ]; then wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover coverage.xml; fi
```

Comment:
This condition is always false now and project will be without code-coverage report.